src/context/websocketContext.jsx

```diff
@@ -96,7 +96,7 @@ export const SocketProvider = (props) => {
 
       if (topic === 'server.restart_requested') setServerRestartingVisible(true)
 
-      if (sender === window.senderId) {
+      if (sender === window.senderId || !sender) {
```

Comment:
You cannot ignore messages without a sender. When server sends message by itself, they don't have a sender ID. This breaks progress bars in bootstrap for example.

Reply:
Removed. Sometimes `activity.created` will have sender=null? 
![image](https://github.com/ynput/ayon-frontend/assets/49156310/6b1e77ba-1c0e-4b24-9e7b-f1e219e45cf1)

Reply:
Yep... In this case sender id was lost (activity is created by a event hook, not a user directly. would it make sense to pass  the sender back? since null is evaluated as "different sender, so I'll invalidate my cache", the worst case scenario is an unnecessary data loading?

Reply:
Cool, I was just checking it wasn't an issue with BE. This actually fixes an issue where the status change activity wouldn't show, but now it does. I might try to get the single activity and patch it into the cache, instead of invalidating and fetching 30 again.